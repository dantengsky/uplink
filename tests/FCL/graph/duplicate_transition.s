transition initial -> a;
transition a -> b;
transition a -> b;
transition b -> terminal;

@initial
start() {
  transitionTo(:a);
}

@a
go() {
  transitionTo(:b);
}

@b
end() {
  terminate("Bye");
}
