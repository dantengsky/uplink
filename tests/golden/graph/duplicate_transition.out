Duplicate definitions found:
 - Duplicate transition: transition a -> b